politeiawww/politeiawww.go

```diff
@@ -131,6 +133,11 @@ type politeiawww struct {
 	// wsDcrdata contains the client and list of current subscriptions to
 	// dcrdata's public subscription websocket
 	wsDcrdata *wsDcrdata
+
+	// The current best block is cached and updated using a websocket
+	// subscription to dcrdata
```

Comment:
Add to this comment that the fallback is the dcrdata best block route if the websocket connection is not active.